pages/product_page.html

```diff
@@ -0,0 +1,361 @@
+                <input type="radio" id="rating" name="rating">
+                <label for="1star">1 Star & above</label>
+              </div>
+    
+        </div>
+
+        <div class="filter-heading">
+            <h4>Sort By</h4>
+        </div>
+            <div class="filter-list">
+            <div>
+                <input type="radio" id="sort" name="sort">
+                <label for="2star">Price - Low to High</label>
+              </div>
+    
+              <div>
+                <input type="radio" id="sort" name="sort">
+                <label for="1star">Price - High to Low</label>
+              </div>
+              </div>
+        </div>
+
+        <div class="doc-content">
+
+            <div class="card card-vertical">
+                <div class="card-main">
+                    <div class="card-header">
+                        <button class="card-btn">
+                            <span>
+                                <i class="far fa-heart"></i>
+                            </span>
+                        </button>
+                        
+                        <img src="/assets/images/product-page-image.jpg" alt=" " class="card-img img-responsive">
+                    </div>
+                     <div class="card-content">
+                        <div class="card-body">
+                                <p class="card-title">Men Suit</p>
+                                <p class="card-subtitle">30 Days Return</p>
+                                <div class="card-price">
+                                    <div class="price">
+                                        <span class="price-discounted">₹54,999.00</span>
+                                        <span class="price-original">₹74,999.00</span>  
+                                    </div>
+                                    <span class="percentage-discount">(27% OFF)</span>  
+                                </div>
+                        </div>
+                        <div class="card-footer">
+                            <button  class="btn btn-primary btn-full-width">
+                                 <span>
+                                    <i class="fas fa-shopping-cart"></i>
+                                </span>
+                                Add to cart
+                            </button>                                   
+                        </div>
+                    </div>   
+                </div>
+            </div>
+
+            <div class="card card-vertical">
+                <div class="card-main">
+                    <div class="card-header">
+                        <button class="card-btn">
+                            <span>
+                                <i class="far fa-heart"></i>
+                            </span>
+                        </button>
+                        
+                        <img src="/assets/images/product-page-image.jpg" alt=" " class="card-img img-responsive">
```

Comment:
You can fill up `alt` attributes of the image tags.

Reply:
updated the alt for images